docs/metadataPackage.md

```diff
@@ -0,0 +1,99 @@
+| parentId                 | yes\*    | The `id` of the object's parent (also within this JSON file).                                                                 |
+| series                   | yes\*    | If the item is a root node in this ingest package set this to the series reference the item should ingest under or `Unknown`. |
+| title                    | no       | The title of this asset.                                                                                                      |
+| transferCompleteDatetime | yes\*\*  | The datetime the transfer to The National Archives was completed.                                                             |
+| transferringBody         | yes\*\*  | The organisation that transferred this record to The National Archives.                                                       |
+| type                     | yes      | The item type (`Asset`).                                                                                                      |
+| upstreamSystem           | yes\*\*  | The system Digital Records Repository received this asset from.                                                               |
+
+\* Either a `parentId` or `series` is required.  
+\*\* Required for custom metadata within the Preservation System.  
+\*\*\* `id_BornDigitalRef`, `id_ConsignmentReference`, and `id_RecordID` are required for custom metadata within the Preservation System.
+
+### File
+
+| Name                 | Required | Description                                                                                                                                                                                                           |
+| -------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| checksum\_\*         | yes      | The hexadecimal checksum(s) for this file in the format. E.g. `checksum_SHA256`. Supported algorithms: `MD5`, `SHA1`, `SHA256`, `SHA512`.                                                                             |
+| fileSize             | yes      | The filesize in bytes.                                                                                                                                                                                                |
+| id                   | yes      | A unique identifier within the ingest package for this file.                                                                                                                                                          |
+| name                 | yes      | The filename.                                                                                                                                                                                                         |
```

Comment:
Should we add "metadata files should end with '-metadata.json'"?

Reply:
Do we enforce that? The only reason they do is because that's what TRE named their metadata file.

Reply:
The validator (that I wrote) is relying on it to distinguish what is a metadata file; should we change this?

Reply:
I think we should. The metadata file is optional and if present should be included within the `originalMetadataFiles` array.